src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -27,6 +27,12 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; 'handler' functions for HTTP endpoints
 
+(defn handle-server-error
+  ([message]
+    (-> (rr/response message)
+        (rr/status 400)
+        (rr/content-type "text/plain"))))
+
```

Comment:
The syntax here is incorrect.  The `[message]` should be outside of the 2nd set of parens.